pandas/tseries/offsets.py

```diff
@@ -576,12 +577,50 @@ def onOffset(self, dt):
 
 
 class BusinessHourMixin(BusinessMixin):
+    start: List[dt_time]
+    end: List[dt_time]
+    n: int
```

Comment:
This is to suppress the mypy errors:
```
"BusinessHourMixin" has no attribute "start"
"BusinessHourMixin" has no attribute "end"
"BusinessHourMixin" has no attribute "n"
```
However this syntax is not compatible with python 3.5...
Any advice?

Reply:
There's a lot of typing stuff only compatible with 3.5.2 onwards. It's an issue we need to address but orthogonal to this PR - is this causing some sort of error here?

Reply:
It's causing `SyntaxError: invalid syntax` in python 3.5.

Reply:
@WillAyd any work around for this problem? Or should I just remove the type hintings?

Reply:
@WillAyd any suggestions?

Reply:
You need to use the “# type: ...” syntax. This syntax is only available for parameters in Python 3.5

Reply:
Thanks for the suggestion. I have removed the function signatures and it's green now.